docs/migration/5_0.md

```diff
@@ -37,6 +37,7 @@ under the section `headerDisplayOnly`.
 - `leaveConfigurationOverview(): void `
 - `goToOrderDetails(owner: CommonConfigurator.Owner): void `
 
-#### Translations (i18n) changes
+#### Translation changes
 
 - `configurator.addToCart.buttonDisplayOnly` was added to `configurator-common.ts`
+- `setAsDefault` text was changed in `address.ts`
```

Comment:
I don't think you want to add this change here. I think these are translation changes related to `configurator add to cart`.

I would suggest:

```
#### ShippingAddress Related Translations (i18n) changes
- Content of `address.addressForm.setAsDefault` changed from `Set as default` to `Set as default shipping address`
```

(I did not verify spelling/punctuation)

Reply:
Changed!